### PR TITLE
Water use parameter colors & loading screen timing for water use

### PR DESCRIPTION
**Pull request by mwernimont**
Before making a pull request
----------------------------
First . . .
- [ ] Clean the code the way Vue likes it - run 'npm run lint --fix'        
- [ ] Make sure all tests run

Then check for accessibly compliance
- [ ] Run WAVE plugin 508 compliance tool

Then run Browserstack; check that application works on . . .
- [x] Chrome
- [x] Safari
- [ ] Edge
- [x] Firefox
- [ ] Samsung Internet
- [ ] Internet Explorer 11 (not supported, but still needs at least a working user redirect page)

Finally . . .
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
Trying to have the loading screen stay up until the map svgs are done loading.  Locally the maps load like lightning so I can't tell if what I did worked, but it loads super slow on test, so guess I have inefficiently test it out there?

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

File: src/views/waterUse/WaterUse.vue [hunk 1]
```diff
@@ -78,23 +78,28 @@
...
         svg: "svg_map_te_spring",
         barchart: "svg_bars_te",
-        useParemeter: "te",
+        useParameter: "te",
         season: "spring"
       }
     },
+    computed: {
+      checkIfSVGMapIsRendered() {
+        console.log("Water Use Function");
```
Comment (mhines-usgs) [inline src/views/waterUse/WaterUse.vue:87]: you sure you want that in there?

File: src/views/waterUse/WaterUse.vue [hunk 2]
```diff
@@ -103,27 +108,24 @@
...
-      },
-      toggleMapInfoBox() {
-        !this.isFirstClick ? this.isAboutMapInfoBoxOpen = !this.isAboutMapInfoBoxOpen : null;
-      },
-      clickAnywhereToCloseMapInfoBox() {
-        this.isAboutMapInfoBoxOpen = !this.isAboutMapInfoBoxOpen;
-        this.isFirstClick = false;
-      },
       addSeasonClass(){
+        console.log("Season class function fired")
```
Comment (mhines-usgs) [inline src/views/waterUse/WaterUse.vue:127]: 👀

File: src/views/waterUse/WaterUse.vue [hunk 3]
```diff
@@ -242,22 +273,22 @@
...
+  stroke: $thermo;
   stroke-linecap: round;
   opacity: 0.6;
 }
 #waterUseBarChartContainer{
   .wu-bars-hover{
     fill-opacity: 0;
     &:hover, &:focus{
-      fill: yellow;
+      fill: rebeccapurple;
```
Comment (mhines-usgs) [inline src/views/waterUse/WaterUse.vue:285]: lol, is that right?

**Comment by mhines-usgs** [comment]
looks good to me locally! seems to load so fast for me locally too, not sure why the server would be so slow.

File: src/views/waterUse/WaterUse.vue (hunk 1 above)
Comment (mwernimont) [reply src/views/waterUse/WaterUse.vue:87]: Currently yes, as I am still testing to see if it actually works.

File: src/views/waterUse/WaterUse.vue (hunk 3 above)
Comment (mwernimont) [reply src/views/waterUse/WaterUse.vue:285]: It is, rebecca gets her own purple I guess.

**Comment by mhines-usgs** [comment]
awe, the story is here: https://en.wikipedia.org/wiki/Eric_A._Meyer

**Pull request merged**